mw-webapp/src/assets/icons/YoutubeIcon.tsx

```diff
@@ -9,6 +9,9 @@ export const YoutubeIcon = (props: IconProps) => {
       xmlns="http://www.w3.org/2000/svg"
       viewBox="0 0 24 24"
       fill="none"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
       className={props.className}
     >
       <path d="M22.54 6.42a2.78 2.78 0 0 0-1.94-2C18.88 4 12 4 12 4s-6.88 0-8.6.46a2.78 2.78 0 0 0-1.94 2A29 29
```

Comment:
How to use CSS var() in .tsx file?

I would remove this `fill` field from this .tsx file, as it doesn't affect anything, because a polygon without `fill` field is transparent by default and the parent (svg) has desirable color. What do you think about it?

Reply:
Удалила свойство